src/tests/evil/evil_test_stdlib.c

```diff
@@ -205,6 +205,34 @@ EFL_START_TEST(evil_stdlib_mkstemp)
 }
 EFL_END_TEST
 
+EFL_START_TEST(evil_stdlib_mkstemp_create_many)
+{
+   const unsigned long long files_to_create = 1000;
+   int fds[files_to_create];
+   const char template[] = "file_XXXXXX";
```

Comment:
Maybe use `getenv("TEMP")` to create files at a temporary directory instead of build?

Reply:
done